manageiq-schema.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "manageiq-password",       ">= 1.2.0", "< 2"
   spec.add_dependency "more_core_extensions",    ">= 3.5", "< 5"
   spec.add_dependency "pg"
-  spec.add_dependency "rails",                   ">=7.0.8.7", "<8.0"
+  spec.add_dependency "rails",                   ">=7.2.2.1", "<8.0"
```

Comment:
Drop 7.0 and 7.1 since we no longer test them and 7.2 is the default for the dummy app, migrations, and rails defaults.